server/src/jobs/assets.h

```diff
@@ -16,27 +16,42 @@
 
 #pragma once
 #include "discovery-task.h"
+#include <set>
 
 // =====================================================================================================================
 
 namespace fty::job {
 
+struct DeviceInfo {
+    int indexDevice;
+    std::string& deviceSerial;
```

Comment:
referenced type in output structure? This is too dangerous.

Reply:
Sorry it is a typo mistake